docs/modules/ROOT/pages/reference/extensions/core.adoc

```diff
@@ -224,6 +245,13 @@ A comma separated list of Ant-path style patterns to match class names that shou
 | `string`
 | 
 
+|icon:lock[title=Fixed at build time] [[quarkus.camel.native.reflection.serialization-enabled]]`link:#quarkus.camel.native.reflection.serialization-enabled[quarkus.camel.native.reflection.serialization-enabled]`
+
+If `true`, basic classes are registered for serialization; otherwise Java serialization will not work in native mode. 
```

Comment:
It would be nice to list the classes here as well. 

> otherwise Java serialization will not work in native mode. 

Isn't that a bit of an overstatement? It would work, but only for the registered classes, no?

Reply:
Add listing classes - I thought that link to the list would be enough, but I can list them here as well.
Yes it is a overstatement, I'll change it.

Reply:
done